packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -5,196 +5,192 @@ $hasExpandButtonLinkLeftPadding: 28px;
 $noExpandButtonLinkLeftPadding: 20px;
 $linkRightPadding: 20px;
 
-:global {
+.root {
+  overflow-y: auto;
+  -webkit-overflow-scrolling: touch;
+  user-select: none;
+}
 
-  .ms-Nav {
-    overflow-y: auto;
-    -webkit-overflow-scrolling: touch;
-    user-select: none;
-  }
+.isOnTop {
+  position: absolute;
+}
 
-  .ms-Nav.is-onTop {
-    position: absolute;
-  }
+.unorderedList {
```

Comment:
If we're going to add class names instead of targeting `ul` and `li` directly, it would be better to use descriptive classes like `.navItems` and `.navItem`.